static/app/views/organizationGroupDetails/groupDetails.tsx

```diff
@@ -609,7 +616,7 @@ class GroupDetails extends Component<Props, State> {
           project={project as Project}
           event={event}
           group={group}
-          replaysCount={replaysCount}
+          replaysCount={undefined}
```

Comment:
I've disabling the count in the tab because it's based on the number of errors that link to replays, but some of those replays might not exist

Before:
<img width="157" alt="Screen Shot 2022-09-06 at 2 42 34 PM" src="https://user-images.githubusercontent.com/187460/188746243-c1e8e474-8750-4644-a2e6-d1f618dce9a1.png">

Reply:
IMO having an incorrect count (bug) is better than having no count (missing feature). At least with the count it'll accomplish its job and getting people to click into the tab.

Reply:
k, but i don't want to take on flak for buggy things

Reply:
Let's open a ticket to figure out the correct count?